src/bin/runners/runner.ts

```diff
@@ -36,12 +36,12 @@ export abstract class Runner {
   protected async executeFlow(): Promise<{ isFailure: boolean; output: FlowResult }> {
     const flowService = Container.get(FlowService);
     const flowResult = await flowService.run();
+    let isFailure = this.archiveArtifactsFailure(flowResult.artifactUploadResults);
+    flowResult.executionResult.forEach(res => {
+      isFailure ||= this.nodeExecutionFailure(res);
+    });
```

Comment:
```suggestion
    return {
      isFailure = this.archiveArtifactsFailure(flowResult.artifactUploadResults) || flowResult.executionResult.find(res => this.nodeExecutionFailure(res)),
      output: flowResult,
    }
```

Reply:
the next return lines should be removed in case you accept this proposal

Reply:
doesn't seem to work. I get a typescript error
```
Type 'true | ExecuteNodeResult[] | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.ts(2322)
runner.ts(36, 44): The expected type comes from property 'isFailure' which is declared here on type '{ isFailure: boolean; output: FlowResult; }'
```